src/templates/sync_page.hdbs

```diff
@@ -3,22 +3,10 @@
   <div class="u-position-relative">
     <!-- Header: Info area -->
     <div class="c-infoarea">
-      <div class="c-infoarea__section u-paddingHorizontal-gutter u-paddingTop-4x u-paddingBottom-4x u-borderBottom-divider">
+      <div class="c-infoarea__section u-paddingHorizontal-gutter u-paddingTop-3x u-paddingBottom-2x u-borderBottom-divider">
         <div class="u-display-flex">
 
-          <div class="u-width-7x u-marginRight-2x">
-            <div class="o-initials-logo u-box-7x u-lineHeight-7x u-fontSize-big">
-              tx
-            </div>
-          </div>
-
-          <div class="u-flexGrow-1 u-color-primary">
-            <div class="u-fontSize-small u-lineHeight-3x">{{project_name}}</div>
-            <div class="u-fontSize-big u-lineHeight-4x">Transifex Sync</div>
-            <div class=" u-color-disabled">
-              <a href="http://docs.transifex.com/integrations/zendesk/" class="o-link u-fontSize-small u-lineHeight-3x" target="_blank">How to use Transifex Sync</a>
-            </div>
-          </div>
+          <div class="u-fontSize-big u-lineHeight-4x u-flexGrow-1 u-color-primary">{{project_name}}</div>
```

Comment:
@rigaspapas I think @samuelchan22 suggestion was to show both brand name and project name, no?